webadmin/fitcrackAPI/src/src/api/fitcrack/endpoints/dictionary/dictionary.py

```diff
@@ -168,6 +168,8 @@ def post(self):
 
             pwd_dist, ret_code = shell_exec([PWD_DIST_PATH, dict_path])
             if ret_code != 0:
+                if os.path.isfile(dict_path):
```

Comment:
fileUpload may throw HTTPException via abort call.. also sorted_cp may throw.. so you need to rework it.

Reply:
When fileUpload throws HTTPException, no file is uploaded. But you are correct with sorted_cp.